src/_includes/amazon-listing-preview.md

```diff
@@ -1,14 +1,14 @@
 
 Your current Amazon listings are compared against your rules, based on the conditions you defined. You can then review which products will move to an ineligible status based on your current Amazon Seller Central account, which products will move from an ineligible state back to an eligible state, and which products will be New Amazon Listings and added to your Amazon listing from your eligible Magento catalog.
 
-Listing Preview allows you to preview your potential Amazon listings and make any necessary adjustments to your listing rules. If you need to adjust your listing rules, click **Listing Rules** in the progress bar to return to the [listing rules]({% link sales-channels/amazon/ob-listing-rules.md %}) step.
+Listing Preview allows you to preview your potential Amazon listings and make any necessary adjustments to your listing rules. If you need to adjust your listing rules, click [**Listing Rules**]({% link sales-channels/asc/listing-rules.md %}) on the store dashboard.
 
-Your potential Amazon listings will populate on the Listing Preview screen in one of three tabs:
+Your potential Amazon listings will populate on the _Listing Preview_ page in one of three tabs:
 
-- **Ineligible Listings**: Products listed on this tab are not eligible for Amazon listing based on your current listing rule settings.
+- **Ineligible Listings**: Products listed are not eligible for Amazon listing based on your current [listing rules]({% link sales-channels/asc/listing-rules.md %}).
 
-   Ineligible products will not be published to Amazon. If an ineligible product is already listed on Amazon and you match the Amazon listing to your Magento catalog product, the quantity for the Amazon listing will change to 0 to prevent sales of the product. To manually remove a listing, see [Ending an Amazon Listing]({% link sales-channels/amazon/end-listings-manually.md %}). Products that are not eligible by Amazon requirements are not listed here. Those products are listed on the [Inactive Listings tab]({% link sales-channels/amazon/inactive-listings.md %}).
+   Ineligible products will not be published to Amazon. If an ineligible product is already listed on Amazon and you match the Amazon listing to your Magento catalog product, the quantity for the Amazon listing will change to `0` to prevent sales of the product. To manually remove a listing, see [Ending an Amazon Listing]({% link sales-channels/asc/end-listings-manually.md %}). Products that are not eligible by Amazon requirements are not listed here. Those products are listed on the [_Inactive Listings_]({% link sales-channels/asc/inactive-listings.md %}) tab.
```

Comment:
```suggestion
   Ineligible products will not be published to Amazon. If an ineligible product is already listed on Amazon and you match the Amazon listing to your Magento catalog product, the quantity for the Amazon listing will change to `0` to prevent sales of the product. To remove a listing manually, see [Ending an Amazon Listing]({% link sales-channels/asc/end-listings-manually.md %}). Products that are not eligible by Amazon requirements are not listed here. Those products are listed on the [_Inactive Listings_]({% link sales-channels/asc/inactive-listings.md %}) tab.
```